scripts/cangen/cangen/can_generator.py

```diff
@@ -220,39 +225,45 @@ def generate_code(bus: Bus, config: Config):
         "bus_name": bus.bus_name,
     }
 
-    logger.debug("Generating code for can messages")
-    _generate_from_jinja2_template(
-        CAN_MESSAGES_TEMPLATE_FILENAME,
-        os.path.join(config.output_dir, bus.bus_name.lower() + CAN_MESSAGES_FILE_NAME),
-        context,
-    )
+    logger.debug("Generating code for can messages and msg registry.")
 
-    logger.debug("Generating code for msg registry")
-    _generate_from_jinja2_template(
-        MSG_REGISTRY_TEMPLATE_FILENAME,
-        os.path.join(config.output_dir, bus.bus_name.lower() + MSG_REGISTRY_FILE_NAME),
-        context,
+    env = _create_jninja_environment(
+        PACKAGE_NAME, {"camel_to_snake": _camel_to_snake, "decimal_to_hex": hex}
     )
+    for template_file_name in TEMPLATE_FILE_NAMES:
+        _generate_jninja_template(
```

Comment:
as with the previous comment, there is so much information to pass into the function that I think we just inline the code.

Also if you chdir into `config.output_dir` (see comment in `generate_code`) then you can modify `_create_output_file_name` to only take the bus and template names